drivers/can/can_max32.c

```diff
@@ -682,7 +682,7 @@ static int can_max32_init(const struct device *dev)
 	return ret;
 }
 
-static const struct can_driver_api can_max32_api = {
+static DEVIC_API(can, can_max32_api) = {
```

Comment:
mmm, nooooope :)
bonus points if you can make it so that CI catches this in the future -- is that driver not built at all in CI?

Reply:
Please be more specific?

Reply:
Ooooooops, well spotted

Reply:
> bonus points if you can make it so that CI catches this in the future -- is that driver not built at all in CI, or was it just somehow not part of the generated testplan for the PR?

I don't think it is built in `tests/drivers/build_all/can`?

Reply:
> Please be more specific?

what do you mean by more specific? The terrible typo spotted above wasn't caught in CI, so something's off is all :)

Reply:
> Shouldn't it be the intention for `build_all` to try to build all of them?

Most of our `build_all` driver tests only build drivers for devices not present/enabled on in-tree boards.